crates/storage-query-datafusion/src/journal/row.rs

```diff
@@ -78,6 +80,13 @@ pub(crate) fn append_journal_row(
                         }
                     }
                 }
+                EnrichedEntryHeader::Sleep { .. } => {
+                    if row.is_sleep_wakeup_at_defined() {
+                        if let Some(sleep_entry) = deserialize_sleep_entry(&entry) {
+                            row.sleep_wakeup_at(sleep_entry.wake_up_time as i64);
```

Comment:
If there is an overflow in the conversion, then I guess we accept it rather than panicking, right?